src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -391,4 +470,28 @@ protected override TokenStreamComponents CreateComponents(string fieldName, Text
             return new TokenStreamComponents(tokenizer, tok);
         }
     }
+
+    public class LuceneStartConfig
```

Comment:
does this need to be public? Do you anticipate clients setting up different configurations?

Reply:
changed to internal